File: Programmers/Inho/Level01/K번째수.cs
using System;
using System.Collections.Generic;
using System.Linq;
public class Solution
{
    public int[] solution(int[] array, int[,] commands)
    {
        int[] answer = new int[] { };
        int[] answerArr = new int[] { };
        List<int> temp = new List<int>();
        List<int> answerList = new List<int>();
        for (int i = 0; i < commands.Length / 3; i++)
        {
            temp.Clear();
            for (int j = commands[i, 0]; j <= commands[i, 1]; j++)
            {
                temp.Add(array[j - 1]);
            }
            temp.Sort();
            answerList.Add(temp.ElementAt(commands[i, 2] - 1));
        }
        return answer = answerList.ToArray();
    }
}

File: Programmers/Inho/C++/Level01/K번째수.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    vector<int> temp;
    // 커맨드 갯수만큼 돌아
    // [i][0]부터 [i][1]포함 범위까지 잘라야함
    // 정렬, ~번째 수 넣기
    for(int i = 0; i<commands.size();i++){
        temp.clear();
        for(int j = commands[i][0];j<=commands[i][1];j++){
            temp.push_back(array[j-1]);
        }
        sort(temp.begin(), temp.end());
        answer.push_back(temp[commands[i][2]-1]);
    }
    return answer;
}